tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drop_participants_without_completed_basics_test.py

```diff
@@ -33,20 +32,17 @@ def setUpClass(cls):
         cls.dataset_id = os.environ.get('COMBINED_DATASET_ID')
```

Comment:
While not completely necessary, it would be good to change this to use the RDR dataset since it is closer to how we plan to execute the rule. 

```suggestion
        cls.dataset_id = os.environ.get('RDR_DATASET_ID')
```